apps/aklite-apps/cmds.cpp

```diff
@@ -172,8 +172,19 @@ int RunCmd::runApps(const std::vector<std::string>& shortlist, const std::string
   auto http_client = std::make_shared<HttpClient>();
   auto docker_client{std::make_shared<Docker::DockerClient>()};
   auto registry_client{std::make_shared<Docker::RegistryClient>(http_client, "")};
-  Docker::RestorableAppEngine app_engine{store_root,    compose_root, docker_root, registry_client,
-                                         docker_client, client,       docker_host, compose_client};
+  Docker::RestorableAppEngine app_engine{
+      store_root,
+      compose_root,
+      docker_root,
+      registry_client,
+      docker_client,
+      client,
+      docker_host,
+      compose_client,
+      Docker::RestorableAppEngine::GetDefStorageSpaceFunc(),
+      [](const Docker::Uri& /* app_uri */, const std::string& image_uri) { return "docker://" + image_uri; },
+      false,
+      true};
```

Comment:
i could have reviewed this in half the time if C++ had named function args :)

Reply:
Probably it's time for introduction of `AppEngineArgs` structure.

Reply:
Its not that big a deal. I just wasn't in a good "C++ headspace" and got the feeling this was changing other stuff - then i  realized it was just there to satisfy the compiler

Reply:
yep, the clang format.